content/pt-br/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Comunicação entre contêineres no mesmo pod usando um volume compartilhado
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+Esta página mostra como usar um Volume para se comunicar entre dois Containers rodando
+no mesmo pod. Veja também como permitir que os processos se comuniquem por
+[compartilhamento do processo de namespace](/docs/tasks/configure-pod-container/share-process-namespace/)
```

Comment:
This is not quite correct. "sharing process namespace" would be more correctly translated as "compartilhamento de namespace do processo".

Reply:
+1